app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/welcomescreen/WelcomeScreen.kt

```diff
@@ -0,0 +1,163 @@
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.sp
+import hr.dice.filipbionda.tmdbpractice.R
+import hr.dice.filipbionda.tmdbpractice.ui.theme.TMDBPracticeTheme
+
+@Composable
+fun WelcomeScreen(modifier: Modifier = Modifier) {
+    Box(
+        modifier = modifier.fillMaxSize(),
+    ) {
+        BackgroundImage()
+        WelcomeScreenContent(modifier = Modifier.align(alignment = Alignment.BottomCenter))
+    }
+}
+
+@Composable
+private fun BackgroundImage(modifier: Modifier = Modifier) {
+    val brush =
+        Brush.verticalGradient(
+            colorStops =
+            arrayOf(
+                0.0f to MaterialTheme.colorScheme.onBackground,
+                0.8f to MaterialTheme.colorScheme.background,
+            ),
+        )
+    Box(
+        modifier = modifier.fillMaxSize()
+    ){
+        Image(
+            painter = painterResource(R.drawable.welcomescreen_background_blurred),
+            contentDescription = null,
+            modifier = Modifier.fillMaxSize(),
+            contentScale = ContentScale.FillBounds,
+        )
+        Canvas(
+            modifier = Modifier.fillMaxSize(),
+        ) {
+            drawRect(
+                brush = brush,
+            )
+        }
+    }
+}
+
+@Composable
+private fun WelcomeScreenContent(modifier: Modifier = Modifier) {
+    Column(
+        modifier = modifier
+            .padding(
+                horizontal = dimensionResource(R.dimen.welcome_screen_content_horizontal_padding),
+                vertical = dimensionResource(R.dimen.welcome_screen_content_vertical_padding),
+            ),
+    ) {
+        val annotatedString =
+            buildAnnotatedString {
+                withStyle(
+                    style = SpanStyle(
```

Comment:
Can you somehow utilize Typography here?

Reply:
Can u explain furthermore?

Reply:
I was thinking that maybe it is possible to use the typography from the Material Theme so it would look something like 
```
 withStyle(style = MaterialTheme.typography.bodyLarge.toSpanStyle())
```
I've used body large as an example it doesn't mean that it is the correct typography

Reply:
Done